models/players.js

```diff
@@ -190,6 +190,15 @@ function compareNFL(a,b) {
 }
 
 
+var formatNBAPlayers = function(response, rb_team_id){
+  var startersarray = response.players.splice(0,5);
+  var bencharray = response.players.splice(5,response.players.length - 5);
```

Comment:
I don't think you need -5 as splice will mutate the original array right. If you do it this way, you should probably use slice.